src/App.jsx

```diff
@@ -5,18 +5,25 @@ import { useEffect, useState } from "react";
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 
-const useSemiPersistentState = () => {
-  const [todoList, setTodoList] = useState(JSON.parse(localStorage.getItem('savedTodoList')) || []);
+const App = () => {
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  
+  useEffect(()=> {
+    new Promise((resolve, reject) => 
```

Comment:
Nit: remove `reject` since you aren't using it, so it becomes `new Promise((resolve) => ...)